contracts/zero-ex/contracts/src/transformers/FillQuoteTransformer.sol

```diff
@@ -161,13 +168,6 @@ contract FillQuoteTransformer is
                 }
             }
 
-            // Ensure we have enough ETH to cover the protocol fee.
```

Comment:
Discussion: If we continue rather than revert we can attempt to handle the gas price adjustment with a fallback. If we revert explicitly early we cannot.